packages/design-tokens/tsconfig.json

```diff
@@ -3,8 +3,17 @@
   "compilerOptions": {
     "resolveJsonModule": true,
     "esModuleInterop": true,
-    "types": ["node", "jest"]
+    "jsx": "react",
+    "types": ["react", "node", "jest"],
+    "paths": {
+      "@metamask/design-system-react": ["../design-system-react/src"]
+    }
```

Comment:
Added path alias for `@metamask/design-system-react` to enable direct imports from the design system package.

### Before
<img width="721" alt="Screenshot 2024-12-02 at 5 05 58 PM" src="https://github.com/user-attachments/assets/44f5e913-28a9-473f-a770-2e4db3b871c6">

### After
<img width="468" alt="Screenshot 2024-12-02 at 5 06 11 PM" src="https://github.com/user-attachments/assets/3e78bbbb-ac4a-4639-945a-1b4057e71317">